src/craftdef.h

```diff
@@ -4,12 +4,12 @@
 
 #pragma once
 
+#include "gamedef.h"
+
 #include <string>
-#include <iostream>
 #include <vector>
-#include <utility>
-#include "gamedef.h"
-#include "inventory.h"
+
+struct ItemStack;
```

Comment:
it doesn't seem that likely that `craftdef.h` would be used anywhere where `ItemStack` is not defined anyway but :shrug: